sdk/cosmos/azure_data_cosmos/src/clients/cosmos_client.rs

```diff
@@ -96,6 +98,34 @@ impl CosmosClient {
         })
     }
 
+    #[allow(dead_code)]
+    async fn initialize(client: CosmosClient) -> azure_core::Result<Self> {
```

Comment:
Let's remove this if we're not calling it yet. We know what we need to do here and can add it in when it's time for it.